docs/data/material/varnish/varnish-extensions/varnish-extensions.md

```diff
@@ -0,0 +1,43 @@
+---
+title: AI2 Varnish Extensions
+githubLabel: 'varnish: varnish-extensions'
+---
+
+# Extensions
```

Comment:
please make this md about varnish-eslint-config, not about extensions in general

Reply:
sure will do!